lib/source-map-from-file.js

```diff
@@ -79,12 +89,22 @@ function sourceMapFromFile (mapURL) {
 // data:[<mediatype>][;base64],<data> see:
 // https://tools.ietf.org/html/rfc2397#section-2
 function sourceMapFromDataUrl (url) {
+  const nodeMajorVersion = Number(process.version.slice(1).split('.')[0])
   const { 0: format, 1: data } = url.split(',')
   const splitFormat = format.split(';')
   const contentType = splitFormat[0]
+  // always evaluates to true
```

Comment:
This could be false I believe, reading the spec:

https://www.rfc-editor.org/rfc/rfc2397#section-2

because base64 is optional. It just happens that our test suite doesn't exercise this case.

Reply:
Thank you for pointing this out.  I changed to comment to reflect the above fact and reference this conversation.